<?xml version="1.0" encoding="UTF-8"?><testrun name="LocacaoServiceTest" project="TestesUnitarios" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="br.ce.wcaquino.servicos.LocacaoServiceTest" time="0.005">
    <testcase name="initializationError" classname="br.ce.wcaquino.servicos.LocacaoServiceTest" time="0.005">
      <error>org.objenesis.ObjenesisException: java.lang.reflect.InvocationTargetException&#13;
	at org.objenesis.instantiator.sun.SunReflectionFactoryHelper.newConstructorForSerialization(SunReflectionFactoryHelper.java:48)&#13;
	at org.objenesis.instantiator.sun.SunReflectionFactoryInstantiator.&lt;init&gt;(SunReflectionFactoryInstantiator.java:41)&#13;
	at org.objenesis.strategy.StdInstantiatorStrategy.newInstantiatorOf(StdInstantiatorStrategy.java:68)&#13;
	at org.objenesis.ObjenesisBase.getInstantiatorOf(ObjenesisBase.java:94)&#13;
	at org.powermock.reflect.internal.WhiteboxImpl.newInstance(WhiteboxImpl.java:250)&#13;
	at org.powermock.reflect.Whitebox.newInstance(Whitebox.java:139)&#13;
	at org.powermock.tests.utils.impl.AbstractTestSuiteChunkerImpl.getPowerMockTestListenersLoadedByASpecificClassLoader(AbstractTestSuiteChunkerImpl.java:95)&#13;
	at org.powermock.modules.junit4.common.internal.impl.JUnit4TestSuiteChunkerImpl.createDelegatorFromClassloader(JUnit4TestSuiteChunkerImpl.java:174)&#13;
	at org.powermock.modules.junit4.common.internal.impl.JUnit4TestSuiteChunkerImpl.createDelegatorFromClassloader(JUnit4TestSuiteChunkerImpl.java:48)&#13;
	at org.powermock.tests.utils.impl.AbstractTestSuiteChunkerImpl.createTestDelegators(AbstractTestSuiteChunkerImpl.java:111)&#13;
	at org.powermock.modules.junit4.common.internal.impl.JUnit4TestSuiteChunkerImpl.&lt;init&gt;(JUnit4TestSuiteChunkerImpl.java:71)&#13;
	at org.powermock.modules.junit4.common.internal.impl.AbstractCommonPowerMockRunner.&lt;init&gt;(AbstractCommonPowerMockRunner.java:32)&#13;
	at org.powermock.modules.junit4.PowerMockRunner.&lt;init&gt;(PowerMockRunner.java:34)&#13;
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)&#13;
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)&#13;
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)&#13;
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)&#13;
	at org.junit.internal.builders.AnnotatedBuilder.buildRunner(AnnotatedBuilder.java:104)&#13;
	at org.junit.internal.builders.AnnotatedBuilder.runnerForClass(AnnotatedBuilder.java:86)&#13;
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#13;
	at org.junit.internal.builders.AllDefaultPossibilitiesBuilder.runnerForClass(AllDefaultPossibilitiesBuilder.java:26)&#13;
	at org.junit.runners.model.RunnerBuilder.safeRunnerForClass(RunnerBuilder.java:59)&#13;
	at org.junit.internal.requests.ClassRequest.getRunner(ClassRequest.java:33)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.createUnfilteredTest(JUnit4TestLoader.java:90)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.createTest(JUnit4TestLoader.java:76)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestLoader.loadTests(JUnit4TestLoader.java:49)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:513)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
Caused by: java.lang.reflect.InvocationTargetException&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at org.objenesis.instantiator.sun.SunReflectionFactoryHelper.newConstructorForSerialization(SunReflectionFactoryHelper.java:44)&#13;
	... 30 more&#13;
Caused by: java.lang.IllegalAccessError: class jdk.internal.reflect.ConstructorAccessorImpl loaded by org.powermock.core.classloader.MockClassLoader @6fc6f14e cannot access jdk/internal/reflect superclass jdk.internal.reflect.MagicAccessorImpl&#13;
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)&#13;
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1012)&#13;
	at org.powermock.core.classloader.MockClassLoader.loadUnmockedClass(MockClassLoader.java:250)&#13;
	at org.powermock.core.classloader.MockClassLoader.loadModifiedClass(MockClassLoader.java:194)&#13;
	at org.powermock.core.classloader.DeferSupportingClassLoader.loadClass1(DeferSupportingClassLoader.java:77)&#13;
	at org.powermock.core.classloader.DeferSupportingClassLoader.loadClass(DeferSupportingClassLoader.java:67)&#13;
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)&#13;
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)&#13;
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1012)&#13;
	at org.powermock.core.classloader.MockClassLoader.loadUnmockedClass(MockClassLoader.java:250)&#13;
	at org.powermock.core.classloader.MockClassLoader.loadModifiedClass(MockClassLoader.java:194)&#13;
	at org.powermock.core.classloader.DeferSupportingClassLoader.loadClass1(DeferSupportingClassLoader.java:77)&#13;
	at org.powermock.core.classloader.DeferSupportingClassLoader.loadClass(DeferSupportingClassLoader.java:67)&#13;
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)&#13;
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)&#13;
	at java.base/java.lang.System$2.defineClass(System.java:2303)&#13;
	at java.base/jdk.internal.reflect.ClassDefiner.defineClass(ClassDefiner.java:66)&#13;
	at java.base/jdk.internal.reflect.MethodAccessorGenerator$1.run(MethodAccessorGenerator.java:401)&#13;
	at java.base/jdk.internal.reflect.MethodAccessorGenerator$1.run(MethodAccessorGenerator.java:395)&#13;
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)&#13;
	at java.base/jdk.internal.reflect.MethodAccessorGenerator.generate(MethodAccessorGenerator.java:394)&#13;
	at java.base/jdk.internal.reflect.MethodAccessorGenerator.generateSerializationConstructor(MethodAccessorGenerator.java:112)&#13;
	at java.base/jdk.internal.reflect.ReflectionFactory.generateConstructor(ReflectionFactory.java:463)&#13;
	at java.base/jdk.internal.reflect.ReflectionFactory.newConstructorForSerialization(ReflectionFactory.java:376)&#13;
	at jdk.unsupported/sun.reflect.ReflectionFactory.newConstructorForSerialization(ReflectionFactory.java:100)&#13;
	... 35 more&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
